src/documentation/runners/apex.md

```diff
@@ -5,5 +5,108 @@ permalink: /documentation/runners/apex/
 ---
 # Using the Apache Apex Runner
 
-This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
+The Apex Runner executes Apache Beam pipelines using [Apache Apex](http://apex.apache.org/) as an underlying engine. The runner has broad support for the [Beam model and supports streaming and batch pipelines]({{ site.baseurl }}/documentation/runners/capability-matrix/).
 
+[Apache Apex](http://apex.apache.org/) is a stream processing platform and framework for low-latency, high-throughput and fault-tolerant analytics applications on Apache Hadoop. Apex has a unified streaming architecture and can be used for real-time and batch processing. With its stateful stream processing architecture, Apex can support all of the concepts in the Beam model (event time, triggers, watermarks etc.).
+
+
+## Apex-Runner prerequisites and setup
+
+To use Apache Apex Runner for Beam, you need to have a Hadoop cluster with YARN setup. You can setup your own Hadoop cluster or simply download DataTorrent Sandbox which comes with Hadoop and Apex pre-installed.
```

Comment:
+1. This new wording gives users what they need to make an informed decision. Maybe a bit less telegraphic. I think it is OK to give generic encouragement to first try it with a distro and only later go a DIY route.

> You can set up your own Hadoop+YARN cluster or use an existing cluster if you have one available. Otherwise, to get started quickly and try out Beam it may be easiest to use a vendor distribution that includes Hadoop and Apex pre-installed. Please see the [distribution information on the Apache Apex website](some link) to learn of options.